### PR TITLE
theme persistence

### PR DESCRIPTION
**Pull request by kungminlin**

### CODE REVIEW

File: assets/js/helpers.js [hunk 1]
```diff
@@ -116,7 +116,9 @@ function updateLists() {
 }
 
 function logout() {
+  var theme = localStorage.theme;
   localStorage.clear();
```
Comment (nayaverdier) [inline assets/js/helpers.js]: Could just remove jwt instead of clearing everything?

File: assets/js/index.js [hunk 2]
```diff
@@ -1,14 +1,18 @@
-server = "https://api.losaltoshacks.com";
+// server = "https://api.losaltoshacks.com";
+server = "http://localhost:5000"
```
Comment (nayaverdier) [inline assets/js/index.js]: Should point to prod

File: assets/js/index.js [hunk 3]
```diff
@@ -1,14 +1,18 @@
...
 
 // Listeners for controls
-let jwt_auth = localStorage.jwt_auth;
-if (window.location.pathname !== "/login.html" && !localStorage.jwt_auth) {
-  window.location.href = "/login.html";
-}
+// let jwt_auth = localStorage.jwt_auth;
+let jwt_auth = "foobar";
```
Comment (nayaverdier) [inline assets/js/index.js]: Revert dev change?

File: assets/js/helpers.js (hunk 1 above)
Comment (JamsheedMistri) [reply assets/js/helpers.js]: Agreed

File: assets/js/index.js (hunk 2 above)
Comment (JamsheedMistri) [reply assets/js/index.js]: We should maybe have this in a gitignored file

File: assets/js/index.js (hunk 3 above)
Comment (JamsheedMistri) [reply assets/js/index.js]: and this

File: assets/js/index.js (hunk 2 above)
Comment (kungminlin) [reply assets/js/index.js]: I don't think it's that significant so gitignore file may not be necessary

**Pull request merged**